fe/fe-core/src/main/java/org/apache/doris/policy/Policy.java

```diff
@@ -91,8 +94,18 @@ public static Policy fromCreateStmt(CreatePolicyStmt stmt) throws AnalysisExcept
                 UserIdentity userIdent = stmt.getUser();
                 userIdent.analyze(ConnectContext.get().getClusterName());
                 TableIf table = db.getTableOrAnalysisException(stmt.getTableName().getTbl());
+                Expression nereidsPredicate = null;
+                if (ConnectContext.get() != null
+                        && ConnectContext.get().getSessionVariable() != null
+                        && ConnectContext.get().getSessionVariable().isEnableNereidsPlanner()) {
```

Comment:
Only need judge `ConnectContext.get().getSessionVariable().isEnableNereidsPlanner()`?

Reply:
ctx could be null in some unit tests